packages/css/src/components/ListItem.css

```
@@ -0,0 +1,81 @@
+.ListItem {
+  padding: var(--space-l);
+  position: relative;
+  display: flex;
+  display: -webkit-flex;
+  flex-direction: row;
+  align-items: center;
+  min-width: 320px;
+}
+
+.ListItem.ListItem--subitem {
```

Here and after you can reduce the selector specificity 

```suggestion
.ListItem--subitem {
```

---

👍